source/index.js

```diff
@@ -175,6 +175,7 @@ module.exports = async (input = 'patch', options) => {
 	tasks.add({
 		title: 'Creating release draft on GitHub',
 		enabled: () => isOnGitHub === true,
+		skip: () => options.releaseDraft,
```

Comment:
Oops 😅 This should have been:

```suggestion
		skip: () => !options.releaseDraft,
```